components/server/src/auth/login-completion-handler.ts

```diff
@@ -44,6 +45,9 @@ export class LoginCompletionHandler {
             await TosFlow.clear(request.session);
             await AuthFlow.clear(request.session);
     
+            if (authHost) {
```

Comment:
I'm wondering why this is optional. Even if it's not necessary to proceed in flow, it would make sense to cary it on. 
todo@alex: make it mandatory